.yarnrc.yml

```diff
@@ -0,0 +1,2 @@
+yarnPath: .yarn/releases/yarn-4.3.1.cjs
+nodeLinker: node-modules
```

Comment:
`\n` police 🚨

Reply:
lol updated in [ad5b9dd](https://github.com/codecov/gazebo/pull/3016/commits/ad5b9ddc57d44448bd2648038482a1d0f8de0b71)